Sources/Plasma/PubUtilLib/plAudio/plWin32Sound.cpp

```diff
@@ -109,7 +109,8 @@ void plWin32Sound::Update()
             uint32_t currentTimeMs = (uint32_t)(GetActualTimeSec() * 1000.0f);
             if (currentTimeMs <= srtReader->GetLastEntryEndTime()) {
                 while (plSrtEntry* nextEntry = srtReader->GetNextEntryStartingBeforeTime(currentTimeMs)) {
-                    if (plgAudioSys::AreSubtitlesEnabled()) {
+                    float currVol = plSound::QueryCurrVolume();
```

Comment:
Does this need the `plSound::` prefix? It's calling a method on the current class (or a parent class), right?

Reply:
I think you're right that it's not necessary, my bad. I'll commit with the prefix removed later today.

Reply:
for the record, this is totally valid C++ and probably does what you want in this case, but it has some non-obvious behaviour... this will always call `plSound`'s `QueryCurrVolume()` method with the current instance, rather than any overridden implementation further down the inheritance chain